### PR TITLE
Keeping default value null when user responsing other than yes or no.

### PR DESCRIPTION
**Pull request by Satendra-SR**
#299 

### Types of changes

- [x] Bug fix (a change which fixes an issue)
- [ ] New feature (a change which adds functionality)

### Short description of what this resolves:

Changed the IVR prompt response "Other" to null which is storing a non-bool value in bool type column.

These changes are required not to store non-boolean data in the bool field. Now if the user responds "Other" for any IVR prompt then null value should store in the database.

### Checklist:

- [x] I have performed a self-review of my own code.
- [x] The code follows the style guidelines of this project.
- [x] The code changes are passing the CI checks
- [ ] I have documented my code wherever required.
- [ ] The changes requires a change to the documentation.
- [ ] I have updated the documentation based on the my changes.

### CODE REVIEW

File: api/models/ivr_prompt_mapping.py [hunk 1]
```diff
@@ -15,4 +15,4 @@ class IvrPromptMapping(TimestampMixin, db.Model):
     ivr_prompt_id = db.Column(db.Integer, db.ForeignKey("ivr_prompt.id"))
     mapped_table_name = db.Column(db.String(255))
     mapped_table_column_name = db.Column(db.String(255))
-    default_value = db.Column(db.String(255))
+    default_value = None
```
Comment (GauravGusain98) [inline api/models/ivr_prompt_mapping.py:18]: How it will work? What is the use of `None` value while defining the model structure

Comment (Satendra-SR) [reply api/models/ivr_prompt_mapping.py:18]: Okay, getting your point. Didn't self reviewed it yet.

File: api/services/prompt_service.py [hunk 2]
```diff
@@ -165,7 +170,8 @@ def update_mapped_fields(
             if class_object_data:
                 setattr(class_object_data, column_name, prompt_response_value)
                 db.session.commit()
-        except:
+        except Exception as e:
+            print(e)
             print("Exception occurred")
```
Comment (GauravGusain98) [inline api/services/prompt_service.py:175]:
```suggestion
            print(f"Exception occurred: {e}")
```

**Review by GauravGusain98 — APPROVED** [review]
Looks good just a minor suggestion

File: api/services/prompt_service.py (hunk 2 above)
Comment (Satendra-SR) [reply api/services/prompt_service.py:175]: Going with the suggested changes.

**Pull request merged**